app/components/doi-related-identifier.js

```diff
@@ -43,13 +43,16 @@ const relationTypeList = [
   'Is original form of',
   'Is identical to',
   'Is collected by',
-  'Collects'
+  'Collects',
```

Comment:
@codycooperross @KellyStathis What order should we follow for this list?

Reply:
In the past, I think this list has been additive, i.e. we just keep adding new values at the bottom :) What are some other options? Alphabetical, according to the list in the schema docs: https://datacite-metadata-schema.readthedocs.io/en/4.5/appendices/appendix-1/relationType/ —any others?